broker-core/src/main/java/io/zeebe/broker/logstreams/state/ZeebeState.java

```diff
@@ -92,4 +96,21 @@ public IncidentState getIncidentState() {
   public KeyGenerator getKeyGenerator() {
     return keyState;
   }
+
+  public BlackList getBlackList() {
```

Comment:
Is this used only for testing? Are the `blacklist` and `isOnBlacklist` methods not enough?